docs/pages/versions/unversioned/sdk/sqlite-next.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: SQLite (Next)
+description: A library that provides access to a database that can be queried through a SQLite API.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-sqlite'
+packageName: 'expo-sqlite'
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+`expo-sqlite` gives your app access to a database that can be queried through a SQLite API. The database is persisted across restarts of your app.
+
+<PlatformsSection android emulator ios simulator />
+
+## Installation
+
+<APIInstallSection />
+
+## Usage
+
+### Importing an existing database
+
+To open a new SQLite database using an existing `.db` file you already have, follow the steps below:
+
+<Step label="1">
+Install `expo-file-system` and `expo-asset` modules:
```

Comment:
```suggestion
Install `expo-file-system` and `expo-asset` libraries:
```

[As per our glossary in writing style guide and page description, let's use library/libraries instead of modules here](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#glossary)